guides/common/modules/proc_verifying-pcp-configuration.adoc

```diff
@@ -25,6 +25,6 @@ Performance Co-Pilot configuration on {foreman-example-com}:
      pmie: primary engine: /var/log/pcp/pmie/{foreman-example-com}/pmie.log
 ----
 
-In this example, both the Performance Metrics Collector Daemon (pmcd), and the Performance Metrics Web Daemon (pmwebd) services are running.
-It also confirms the PMDAs which are collecting metrics.
-Finally, it lists the currently actively archive file, in which `pmlogger` is storing metrics.
+In this example, both the Performance Metrics Collector Daemon (pmcd) and Performance Metrics Web Daemon (pmwebd) services are running.
```

Comment:
@evgeni FYI, this is a bit confusing, because `pmwebd` is not in the example output above. Are we missing a piece of the output? If not, then do we have to mention this daemon, when it isn't in the example?

Reply:
Huh, this one did slip in. Yeah, pmwebd is not a thing anymore in PCP5+, let me see if we need to mention proxy or nothing.

Reply:
Yeah, it should be the proxy. So I guess we should write "Proxy for Performance Metrics Collector Daemon (pmproxy)" or "Performance Metrics Proxy Daemon (pmproxy)" or something similar (that wording is taken from their own docs: https://man7.org/linux/man-pages/man1/pmproxy.1.html https://pcp.readthedocs.io/en/latest/QG/QuickReferenceGuide.html#pcp-web-services )

Reply:
Perfect, thanks!